### PR TITLE
Dapps over Apps

### PR DESCRIPTION
**Pull request by supercoolkay**
Updated Celo Dapp over Apps proposal

### CODE REVIEW

File: CGPs/cgp-0107.md [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+* Workshop & Training for developers & non- developers on how to build with Celo. Kindly check out some of our technical workshops videos:
+* Organize startup pitch competition for project founders looking to build or integrate Celo’s infrastructure to their projects/start-ups.Kindly check out the one we hosted before
+* Organize hackathons and incubation programs focused on providing solutions to real life problems applicable to Africa and facing the world at large.
+* Guide project building on Celo to MVP, testnet and mainnet respectively.
+* Establish blockchain clubs in various institutions in Africa.
+* Create visibility via content creation and distribution.
+* Collaborate and consolidate efforts with various institutions and communities aligned with our goal.
+* Personal mentorship program with project founders, developers, tech enthusiasts and web3 newbies looking forward to building with Celo.
+
+|Particulars|KPIs & Deliverables|Budget-$63,400|Duration|
```
Comment (0xzoz) [inline CGPs/cgp-0107.md:67]: Please add the Celo amount as the governace proposal payload is in CELO which fluctuates

File: CGPs/cgp-0107.md [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+|**Operational Cost - Less than 10% of budget**|4 Core team members will work for 20 hours per week|**Ask: $5,500**|6 months|
+
+Social Media:
+
+* [Dapps over Apps on X (twitter) ](https://www.twitter.com/dappsoverapps.com/)
+* [Telegram ](https://t.me/+OrxSivWwoh1jZDU0)
+* [Dappsoverapps@gmail.com](mailto:Dappsoverapps@gmail.com)
+* [Instagram](https://www.instagram.com/yestodapps)
+
+Multisig:
```
Comment (0xzoz) [inline CGPs/cgp-0107.md:124]: Please add the multisig address(`0x79d4881D34B6159aBcAb35Ad5029a4DE66c28626`) which is in the payload for visibility.

Comment (supercoolkay) [reply CGPs/cgp-0107.md:124]: Hi @0xzoz Thank you for your time and patience. Corrections have been made accordingly. Thanks once again!

**Comment by supercoolkay** [comment]
Hello @0xzoz @juanjgiraldoc . Goodday, I would like to call your attention to our pull request. Thank you!

**Comment by 0xzoz** [comment]
@supercoolkay i wanted to provide some context here to why your PR has not been approved as yet.

Celo is currently going through a governance restructure and it is advised to hold current proposals due to this not being formalized.

Most members of the community and ecosystem are voluntarily taking part and acknowledging the guidance. That being said, it is not mandatory but may affect participation in voting and ultimately the result of a submitted proposal. 

If you would like to read more about the proposed changes check out the forum posts by Luuk

**Review by 0xzoz — APPROVED** [review]

**Review by juanjgiraldoc — APPROVED** [review]

**Comment by supercoolkay** [comment]
Hello @0xzoz. Goodday. Thanks for the support and guidance so far.

Our proposal is currently on draft. I would like to ask if it can be live for voting now. 
Thanks once again for your time.

**Comment by 0xzoz** [comment]
Hey there @supercoolkay 

The protocol is permissionless and you have taken the necessary steps prior to submission. The timing of when you would like to submit your proposal is entirely up to your team. It requires nothing on our end. 

If you need information on submitting a proposal via the CLI you can view that [here](https://github.com/celo-org/governance/blob/main/Technical%20Guide%20-%20Nuts%20%26%20Bolts%20of%20the%20CGP%20Process.md)

**Comment by supercoolkay** [comment]
Hi @0xzoz,

Thank you so much for always taking out time to respond to our concerns. It is live now. Your support has been really helpful! We are excited to be a part of Celo.

Many thanks! 🫡❤️

**Comment by supercoolkay** [comment]
Hello @0xzoz. I hope you had a good time during your holiday? We submitted our proposal for Celo Dapps over Apps last Monday but it did not pass for voting.

We would like to know when the governance council will resume office, probably it could be voted on before the year ends. 

Thank you.

**Pull request merged**